frontend/src/components/admin/NewMinespaceUser.js

```diff
@@ -35,6 +35,17 @@ export class NewMinespaceUser extends Component {
     });
   };
 
+  handleSearch = (name) => {
+    if (name.length > 2) {
+      // Only used by the map view, which searches by name
```

Comment:
This comment is incorrect here. It's only relevant to the implementation of `handleSearch` in the frontend dashboard.

Reply:
Yea, that was a copy pasta error from the MineSearch (there i took this snippet from).